.gitignore

```diff
@@ -103,4 +103,5 @@ ENV/
 data/
 .DS_Store
 .virtualenvs/
-test.env
\ No newline at end of file
+test.env
+run_tests_single.sh
```

Comment:
I don't see this file

Reply:
Yeah, I put it in the gitignore so it didn't get checked in accidentally. It's just a hack to `run_tests.sh` to allow running a single file of tests